config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -66,6 +66,7 @@ presubmits:
       preset-k8s-ssh: "true"
       preset-bazel-scratch-dir: "true"
       preset-e2e-scalability-common: "true"
+      preset-e2e-scalability-containerd: "true"
```

Comment:
This is not 5k presubmit - please update PR description.

Reply:
It's also optional, so we won't get any gain from it...

Reply:
> This is not 5k presubmit - please update PR description.

Done.

> It's also optional, so we won't get any gain from it...

I'm not sure what do you mean by that,

Are you suggesting that this preset should not be rolled out to this test? Or maybe that we can safely remove this presubmit?

Reply:
I'm saying "rolling this out has zero value, because noone is using it".

Reply:
[I'm not saying "don't do that" - I'm fine with that. I'm saying that this is effectively "no-op".]

Reply:
Sure, thanks for explanation.

Then I would like to merge this change to keep this config in line with other configs. If there will be an event that some will use this presubmit, then it should be configured in an expected way.